testsuite/forge-cli/src/main.rs

```diff
@@ -1052,6 +1066,48 @@ fn changing_working_quorum_test(
         )
 }
 
+fn large_db_test(
+    num_validators: usize,
+    target_tps: usize,
+    min_avg_tps: usize,
+    existing_db_tag: String,
+) -> ForgeConfig<'static> {
+    let config = ForgeConfig::default();
+    config
+        .with_initial_validator_count(NonZeroUsize::new(num_validators).unwrap())
+        .with_initial_fullnode_count(std::cmp::max(2, target_tps / 1000))
+        .with_network_tests(vec![&PerformanceBenchmark])
+        .with_existing_db(existing_db_tag.clone())
+        .with_node_helm_config_fn(Arc::new(move |helm_values| {
+            helm_values["validator"]["storage"]["labels"]["tag"] = existing_db_tag.clone().into();
+            helm_values["fullnode"]["storage"]["labels"]["tag"] = existing_db_tag.clone().into();
+            helm_values["validator"]["config"]["base"]["working_dir"] =
```

Comment:
Do you mean `data_dir`? https://github.com/aptos-labs/aptos-core/blob/1a19d0a5704e1198a90b3b5996b3385f0c0ada7a/config/src/config/mod.rs#L111 I don't see any reference to `working_dir`

Reply:
It was added in https://github.com/aptos-labs/aptos-core/pull/4591/files